src/pages/books.js

```diff
@@ -1,27 +1,26 @@
-import React, { useState } from 'react';
-import BookList from '../components/bookList';
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { addBook, removeBook } from '../redux/books/bookSlice';
+import BookDisplay from '../components/booksDisplay';
 import BookForm from '../components/bookForm';
 
 function Books() {
-  const [books, setBooks] = useState([
-    { id: 1, title: 'Book 1', author: 'Author 1' },
-    { id: 2, title: 'Book 2', author: 'Author 2' },
-  ]);
+  const books = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
 
-  const handleDeleteBook = (bookId) => {
-    const updatedBooks = books.filter((book) => book.id !== bookId);
-    setBooks(updatedBooks);
+  const handleAddBook = (newBook) => {
+    dispatch(addBook(newBook));
   };
```

Comment:
- [ ] Kindly fix the following behavior: when removing a book only the first one is removed each time. (See GIF attached below)

![chrome-capture-2023-9-2](https://github.com/richie1988/My_Bookstore_App/assets/62290545/dd839226-24b3-42b8-ba7b-685576b834db)